### ISSUE TITLE
Default imports with class types and class-based React components are always appearing styled as classes in latest Insiders

### ISSUE THREAD

**Issue by iansan5653** [microsoft/vscode#88696]
Version: 1.42.0-insider (user setup)
Commit: b40e5ce729ce49b7311e92f65f1f21c514aa9c2c
Date: 2020-01-15T05:40:15.219Z
Electron: 7.1.7
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.16299

I'm not sure if this is a VSCode issue or an extension, but starting today when I updated to 1.42.0, all default imports of classes, and all TSX tags representing class-based React components, are appearing as green with an underline, a style typically reserved for classes. Before this, tags were italic blue and all imports were white.

Technically the TSX tags do represent class instances, but it makes it very difficult to tell them apart from the component props and they look very strange next to functional components.

Imports example: 
![image](https://user-images.githubusercontent.com/2294248/72454572-1cb9a180-378f-11ea-8f68-163053a2e060.png)

TSX example:
![image](https://user-images.githubusercontent.com/2294248/72454741-6e622c00-378f-11ea-937a-4c219d2e1e72.png)

This doesn't happen with the initial load of the file, only after the TS language server starts.

This does still happen with extensions disabled.

**Comment by iansan5653** [comment]
Seems to be resolved after updating.

Version: 1.42.0-insider (user setup)
Commit: 33c79d5ad447956814a2a3658029dffb9e28bae6
Date: 2020-01-15T09:36:35.018Z
Electron: 7.1.7
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.16299

**Comment by iansan5653** [comment]
Never mind, it's still happening.

**Comment by mjbvz** [comment]
Likely caused by semantic highlighting. Try setting `"editor.semanticHighlighting.enabled": false` to disable it

**Comment by aeschli** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/88911
